apps/backend/alembic/versions/f4a2b1c8d37e_create_unified_infrastructure_schema.py

```
@@ -0,0 +1,337 @@
+"""Create unified infrastructure schema with TimescaleDB compatibility
+
+Revision ID: f4a2b1c8d37e
+Revises: 
+Create Date: 2025-08-04 01:10:00.000000
+
+"""
+from typing import Sequence, Union
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)